client/src/App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import { Route } from 'react-router-dom';
+
+import Modal from './components/UI/Modal/Modal';
+import './App.css';
+import Layout from './components/hoc/Layout/Layout';
+import Home from './containers/Home';
+
+const App = () => {
+  const [showAuthModal, setShowAuthModal] = useState(true);
+
+  return (
+    <>
+      <Layout>
+        <Route
+          path="/signin"
+          render={props => {
+            setShowAuthModal(true);
+            return (
+              <Modal
+                history={props.history}
+                show={showAuthModal}
+                modalClosed={() => {
+                  setShowAuthModal(false);
+                }}
+              >
+                <h1> login page</h1>
+              </Modal>
+            );
+          }}
+        />
+        <Route path="/" component={Home} />
```

Comment:
I would suggest creating an array containing routes and its components like:

```
[
{
 name: 'HOME',
path: '/home',
component:HOME
},
...
]
```
This will take our number of routes to another file. :smile: